prow/config.yaml

```diff
@@ -19,35 +19,35 @@ presubmits:
     always_run: true
     context: pull-cadvisor-e2e
     rerun_command: "@k8s-bot pull-cadvisor-e2e test this"
-    trigger: "@k8s-bot (pull-cadvisor-e2e )?test this"
+    trigger: "(^@k8s-bot (pull-cadvisor-e2e )?test this$|^/test( pull-cadvisor-e2e)?$)"
```

Comment:
Most REs seem to have a `\r?$` pattern, not sure why (ask @spxtr) maybe something about windows?

https://github.com/kubernetes/test-infra/blob/master/prow/plugins/assign/assign.go#L33